services/src/main/java/org/keycloak/protocol/oidc/mappers/UserAttributeMapper.java

```diff
@@ -39,7 +39,7 @@
  * @author <a href="mailto:bill@burkecentral.com">Bill Burke</a>
  * @version $Revision: 1 $
  */
-public class UserAttributeMapper extends AbstractOIDCProtocolMapper implements OIDCAccessTokenMapper, OIDCIDTokenMapper {
+public class UserAttributeMapper extends AbstractOIDCProtocolMapper implements OIDCAccessTokenMapper, OIDCIDTokenMapper, UserInfoTokenMapper {
```

Comment:
Currently I only support userinfo configuration for `UserAttributeMapper`, which other mappers should also provide this support and which not?
Seems like the code for transforming / filtering / injecting claims is often very similar - this could also be streamlined a bit.